cypress/support/hostsfunctions.ts

```diff
@@ -322,3 +335,30 @@ export function checkFactsInHost(inventory: Inventory, hostInInventory?: boolean
   cy.containsBy('a', 'Facts').click();
   cy.get('code').should('contain', 'ansible_dns');
 }
+
+export function testMissingButton(
+  host_type: string,
+  inventory: Inventory,
+  //view: string,
```

Comment:
Is this commented out to be fixed in another PR? If not, we should delete it.

Reply:
done